cfgov/legacy/views.py

```diff
@@ -13,9 +21,6 @@ def get_render_url(self):
     def get(self, request):
 
         self.request = request
-        if settings.DEBUG and PDFReactor is None:
-            return HttpResponse("PDF Reactor is not configured, can not render %s" % self.get_render_url())
```

Comment:
yeah the capitalization was causing errors, thought it was related to pdfReactor not loading correctly. i've put it back in,  ~_~ its been a slow morning.